slick.grid.js

```diff
@@ -3120,7 +3128,16 @@ if (typeof Slick === "undefined") {
       addlCssClasses += (formatterResult && formatterResult.addClasses ? (addlCssClasses ? ' ' : '') + formatterResult.addClasses : '');
       var toolTip = formatterResult && formatterResult.toolTip ? "title='" + formatterResult.toolTip + "'" : '';
 
-      stringArray.push("<div class='" + cellCss + (addlCssClasses ? ' ' + addlCssClasses : '') + "' " + toolTip + ">");
+      var customAttrStr = '';
+      if(m.hasOwnProperty('cellAttrs') && m.cellAttrs instanceof Object) {
```

Comment:
you've put `m.cellAttrs instanceof Object` but I would rather see `m.cellAttrs instanceof Array` as I suggested originally.

Reply:
Can you change this one though.. Thanks

Reply:
It's not an Array, it's an object like this:
```
                    cellAttrs: {
                        'col-id': val.id
                    }

```

Reply:
oh dang my bad again, I thought you were looping through an array of attributes but instead you're looping on an array of objects. I see why you're using the `for in` now 👍